Services/Init/classes/class.ilErrorHandling.php

```diff
@@ -356,8 +358,28 @@ protected function isDevmodeActive() {
 	protected function defaultHandler() {
 		// php7-todo : alex, 1.3.2016: Exception -> Throwable, please check
 		return new CallbackHandler(function($exception, Inspector $inspector, Run $run) {
+			global $lng;
+			$lng->loadLanguageModule('logging');
```

Comment:
Das ErrorHandling wird sehr früh initialisiert.

bash>phpunit Services/AccessControl/test/ilServicesAccessControlSuite.php führt zu 
PHP Fatal error:  Call to a member function loadLanguageModule()